assets/src/scripts/components/hydrograph/drawing-data.js

```diff
@@ -220,6 +221,36 @@ export const getCurrentVariableMedianStatPoints = createSelector(
         });
     });
 
+/*
+ * @ return {Array of Arrays of Objects} where the properties are date (universal), and value
+*/
+export const getWaterwatchFloodLevelDataPoints = createSelector(
```

Comment:
See the comment on the time-series-graph. I'm thinking this selector might not be needed at all. You would just use the getWaterwatchFloodLevels directly.

Reply:
I removed this selector from drawing-data.